tracing-subscriber/src/macros.rs

```diff
@@ -0,0 +1,24 @@
+macro_rules! try_lock {
+    ($lock:expr) => {
+        try_lock!($lock, else return)
+    };
+    ($lock:expr, else $els:expr) => {
+        if let Ok(l) = $lock {
+            l
+        } else if std::thread::panicking() {
+            $els
+        } else {
+            panic!("lock poisoned")
+        }
+    };
+}
+
+macro_rules! cfg_feature {
+    ($name:expr, { $($item:item)* }) => {
```

Comment:
nit/TIOLI: i _think_ `$name` could be a `literal` here, AFAIK it's always going to be a string literal?
```suggestion
    ($name:literal, { $($item:item)* }) => {
```
(cc @jyn514, am I correct?)

Reply:
Sure, if it's always a string literal you could use `literal`. https://veykril.github.io/tlborm/macros/minutiae/fragment-specifiers.html?highlight=literal#literal

Reply:
that's what i thought --- i couldn't _think_ of a case where a feature name could be anything other than a string literal but thought it was worth asking.

Reply:
You can always change the macro if you need to something more complicated like `cfg(unix)`, but yeah I think for the current setup it will always be a string.

Reply:
That's true, it's purely internal.